src/commands/createFunction/durableSteps/netherite/NetheriteConfigureHostStep.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from '@microsoft/vscode-azext-azureappservice/out/src/extensionVariables';
+import { AzExtFsExtra, AzureWizardExecuteStep, nonNullValue } from '@microsoft/vscode-azext-utils';
+import * as path from 'path';
+import { hostFileName } from '../../../../constants';
+import { hostJsonConfigFailed } from '../../../../constants-nls';
+import { IHostJsonV2, INetheriteTaskJson } from '../../../../funcConfig/host';
+import { netheriteUtils } from '../../../../utils/durableUtils';
+import { IEventHubsConnectionWizardContext } from '../../../appSettings/IEventHubsConnectionWizardContext';
+
+export class NetheriteConfigureHostStep<T extends IEventHubsConnectionWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 250;
+
+    public async execute(context: T): Promise<void> {
+        try {
+            const hostJsonPath: string = path.join(context.projectPath, hostFileName);
+            const hostJson: IHostJsonV2 = await AzExtFsExtra.readJSON(hostJsonPath) as IHostJsonV2;
+
+            const durableTask = hostJson.extensions?.durableTask as INetheriteTaskJson ?? {};
+            const existingHubName: string | undefined = durableTask?.hubName;
+
+            hostJson.extensions ??= {};
+            hostJson.extensions.durableTask = netheriteUtils.getDefaultNetheriteTaskConfig(
+                nonNullValue(context.newEventHubName || existingHubName)
+            );
+
+            await AzExtFsExtra.writeJSON(hostJsonPath, hostJson);
+        } catch {
+            ext.outputChannel.appendLog(hostJsonConfigFailed);
```

Comment:
I think it would be helpful to output the error message as well. 

For example, if the error is just a JSON parse failed error, it'd be a lot easier for the user to diagnose and resolve.

It could just be in the output channel as well. Though, I think that the warning message itself should appear as a notification with a button that leads to the output channel for more information.

Their project straight up won't work if this fails, correct? So we should make sure the user notices this warning.

Reply:
You bring up a good point, this is more of a critical error since the project won't work properly.  I'm thinking instead of logging it in the channel at all, I could just take out the try/catch and have the wizard throw the error instead.  How do you feel about that solution?